nmdc_runtime/site/translation/neon_benthic_translator.py

```diff
@@ -0,0 +1,550 @@
+import re
+import sqlite3
+
+import pandas as pd
+import requests
+import requests_cache
+
+from nmdc_schema import nmdc
+from nmdc_runtime.site.translation.translator import Translator
+from nmdc_runtime.site.util import get_basename
+from nmdc_runtime.site.translation.neon_utils import _get_value_or_none, _create_controlled_identified_term_value, _create_controlled_term_value, _create_geolocation_value, _create_quantity_value, _create_timestamp_value, _create_text_value
+
+
+BENTHIC_BROAD_SCALE_MAPPINGS = {
+    "stream": {"term_id": "ENVO:01000253", "term_name": "freshwater river biome"}
+}
+
+BENTHIC_LOCAL_SCALE_MAPPINGS = {
+    "pool": {"term_id": "ENVO:03600094", "term_name": "stream pool"},
+    "run": {"term_id": "ENVO:03600095", "term_name": "stream run"},
+    "step pool": {"term_id": "ENVO:03600096", "term_name": "step pool"},
+    "riffle": {"term_id": "ENVO:00000148", "term_name": "riffle"},
+    "stepPool": {"term_id": "ENVO:03600096", "term_name": "step pool"},
+}
+
+BENTHIC_ENV_MEDIUM_MAPPINGS = {
+    "plant-associated": {
+        "term_id": "ENVO:01001057",
+        "term_name": "environment associated with a plant part or small plant",
+    },
+    "sediment": {"term_id": "ENVO:00002007", "term_name": "sediment"},
+    "biofilm": {"term_id": "ENVO:00002034", "term_name": "biofilm"},
+}
+
+
+class NeonBenthicDataTranslator(Translator):
+    def __init__(self, benthic_data: dict, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+        self.conn = sqlite3.connect("neon.db")
```

Comment:
I'm still concerned about writing this to disk and now we have two `Translator` classes that are accessing the same file! I'm a little worried about cross-contamination between runs of different translators.